Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -2376,6 +2379,19 @@ final class KSRAnalyticsTests: TestCase {
     XCTAssertEqual(true, props?["discover_recommended"] as? Bool)
     XCTAssertEqual("recs_home", props?["discover_ref_tag"] as? String)
   }
+
+  /*
+   Helper to test logged in User properties from a template User
+   */
+  func assertUserProperties(_ props: [String: Any]?) {
```

Comment:
So since it's just in one function that you need this properties, you can just add it directly to the function and duplicate for DataLake or Segment. I think it would be evident when we are removing DataLake

Reply:
Yeah I think we can avoid writing a method for this since we are only asserting user properties in one particular place.